README.md

```diff
@@ -1,2 +1,51 @@
 # in-stock-bot
-A simple python telegram bot checking if a product is in stock
+* [![GitHub Stars](https://img.shields.io/github/stars/MayNiklas/in-stock-bot.svg)](stargazers)
+
+* [![GitHub Issues](https://img.shields.io/github/issues/MayNiklas/in-stock-bot.svg)](issues)
+
+* [![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+* [![Telegram Chat](https://img.shields.io/badge/Telegram-Join%20to%20us-blue.svg)](TelegramChannel)
+
+A small python Bot with Telegram notifications for checking availability of a product.
+
+![Chat Preview](./inStockBot.jpg)
+
+## Setup
+Clone this repo to your enviroment and change the `config.json` at first.
+In the `config.json` your must add your Telegram Api Key and one or multiple products.
+```json
```

Comment:
Very good explanation of what to do!

BUT: maybe we should talk about how we want to implement the "multi product changes" first, since this config isn't merged to the main branch yet.

Reply:
Then merg the quickly pure ;)

Reply:
Or I can take out the documentation part for now, and adjust / insert it after we have decided on a solution.

Reply:
@EinfachAleks I think I would keep it in as a placeholder  🤔